src/components/Dialogs/Multistepper/Multistepper.style.ts

```diff
@@ -0,0 +1,94 @@
+import styled from '@emotion/styled'
+import Dialog from '../GeneralDialog/GeneralDialog'
+import { Text, Button } from '@/shared/components'
+import { colors, sizes, breakpoints, typography } from '@/shared/theme'
+
+type CircleProps = {
+  isFilled?: boolean
+}
+
+type StyledStepInfoProps = {
+  isActive?: boolean
+}
+
+export const StyledDialog = styled(Dialog)`
+  max-width: 700px;
+`
+
+export const StyledHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  border-bottom: 1px solid ${colors.gray[500]};
+  margin: 0 calc(-1 * var(--dialog-padding-small));
+  padding: 0 var(--dialog-padding-small);
+  padding-bottom: var(--dialog-padding-small);
```

Comment:
Why is this based on dialog padding? I think this component can specify this padding by itself